templates/prometheus.go

```diff
@@ -31,7 +25,7 @@ var (
 var PrometheusSpecTemplate = promv1.PrometheusSpec{
 	CommonPrometheusFields: promv1.CommonPrometheusFields{
 		ServiceAccountName:     "prometheus-k8s",
-		ServiceMonitorSelector: &serviceMonitorSelector,
+		ServiceMonitorSelector: &metav1.LabelSelector{},
```

Comment:
Why select all the ServiceMonitors? We already have labels that is used by OCP monitoring. We could re-use the same here.

Reply:
@umangachapagain can you be more specific. Which labels? I can see all the service monitors have different labels

Reply:
ok, I re-checked. OCP is using NamespaceSelector with empty ServiceMonitorSelector. 
This implementation is fine.